WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -193,16 +193,13 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         }
 
         view.addSubview(scrollView)
+        view.pinSubviewToAllEdges(scrollView)
```

Comment:
Turns out we need to pin the scroll view to the superview, NOT the safe area, in order for it to play nicely with all the inset logic in BlogDetailsVC